CHANGES.txt

```diff
@@ -1,3 +1,77 @@
+  * Removed the internal-only header coded_stream_inl.h and the internal-only methods defined there.
+  * Enforced no SWIG wrapping of descriptor_database.h (other headers already had this restriction).
+  * Implementation of the equivalent of the MOMI parser for serialization. This removes one of the two serialization routines, by making the fast array serialization routine completely general. SerializeToCodedStream can now be implemented in terms of the much much faster array serialization. The array serialization regresses slightly, but when array serialization is not possible this wins big. 
+  * Do not convert unknown field name to snake case to accurately report error.
+  * Fix a UBSAN warnings. (#6333)
+  * Add podspec for C++ (#6404)
+  * protoc: fix source code info location for missing label (#6436)
+  * C++ Add move constructor for Reflection's SetString (#6477)
+
+  Java
+  * Call loadDescriptor outside of synchronized block to remove one possible source of deadlock.
+  * Have oneof enums implement a separate interface (other than EnumLite) for clarity.
+  * Opensource Android Memory Accessors
+  * Update TextFormat to make use of the new TypeRegistry.
+  * Support getFieldBuilder and getRepeatedFieldBuilder in ExtendableBuilder
+  * Add proguard config generator for YouTubeBenchmarkSuiteLite.
+  * Update JsonFormat to make use of the new TypeRegistry.
+  * Add proguard config generator for GmmBenchmarkSuiteLite.
+  * Change ProtobufArrayList to use Object[] instead of ArrayList for 5-10% faster parsing
+  * Implement ProtobufArrayList.add(E) for 20% (5%-40%) faster overall protolite2 parsing
+  * Make a copy of JsonFormat.TypeRegistry at the protobuf top level package. This will eventually replace JsonFormat.TypeRegistry.
+  * Fix javadoc warnings in generated files (#6231)
+  * Java: Add Automatic-Module-Name entries to the Manifest (#6568)
+
+  Python
+  * Add descriptor methods in descriptor_pool are deprecated.
+  * Uses explicit imports to prevent multithread test failures in py3.
+  * Added __delitem__ for Python extension dict
+  * Update six version to 1.12.0 and fix legacy_create_init issue (#6391)
+
+  JavaScript
+  * Remove deprecated boolean option to getResultBase64String().
+  * Cast alphabet type properly to base64.Alphabet
+  * Add Alphabet enum to getResultBase64String().
+    
+    First step in migrating getResultBase64String away from deprecated boolean option.
+    
+    (a) this aligns with other base64 API's
+    (b) this removes the need for some clients that strip padding from web safe
+    (c) allows new API for JsProto to use the enum rather than limited boolean options.
+  * Delete jspb.BinaryIterator.
+  * Fix sint64 zig-zag encoding.
+  * Simplify hash64 string conversion to avoid DIGIT array. Should reduce overhead if these functions aren't used, and be more efficient by avoiding linear array searches.
+  * Use jasmine expect instead of closure test asserts.
```

Comment:
not needed

Reply:
Done

Reply:
Actually, do you mean all four of these are not needed or just "Delete jspb.BinaryInterator" is not needed?